web/beacon-app/src/features/onboarding/components/steps/workspace/form.tsx

```diff
@@ -26,6 +29,13 @@ const WorkspaceForm = ({ onSubmit, isSubmitting }: WorkspaceFormProps) => {
     };
   }, [touched.workspace, setFieldValue, values, touched]);
 
+  // set error if workspace is already taken
+  useEffect(() => {
+    if (hasError) {
+      setFieldError('workspace', t`This workspace is already taken.`);
```

Comment:
I saw this error message in Figma. `The workspace URL is taken by another team. Try a variation or another slug.`

Although most users will try another slug, this message should help give them some direction on what to do next.

Reply:
Well found.  I forgot to update the copy with what was added in Figma. Thanks